lib/mpl_toolkits/mplot3d/proj3d.py

```diff
@@ -51,34 +51,40 @@ def world_transformation(xmin, xmax,
                      [0,    0,    0,    1]])
 
 
-def view_transformation(E, R, V):
+def rotation_about_vector(v, angle):
+    """
+    Produce a rotation matrix for an angle in radians about a vector.
+    """
+    v = v/np.linalg.norm(v)
+    s = np.sin(angle)
+    c = np.cos(angle)
+    t = 1-c
```

Comment:
Not a numerical expert, but https://math.stackexchange.com/a/142831 claims that using 2*sin²(angle/2) is more numerically stable.